easybuild/easyblocks/c/cuda.py

```diff
@@ -136,8 +136,25 @@ def create_wrapper(wrapper_name, wrapper_comp):
         for comp in (self.cfg['host_compilers'] or []):
             create_wrapper('nvcc_%s' % comp, comp)
 
+        ldconfig = which('ldconfig')
+        if ldconfig:
+            self.log.info("ldconfig found at %s", ldconfig)
+        else:
+            # ldconfig is usually in /sbin or /usr/sbin
+            for cand_path in ['/sbin', '/usr/sbin']:
+                if os.path.exists(os.path.join(cand_path, 'ldconfig')):
+                    ldconfig = os.path.join(cand_path, 'ldconfig')
+                    self.log.info("ldconfig found at %s." % ldconfig)
+                    break
+
+        # fail if we couldn't find ldconfig, because it's really needed
+        if not ldconfig:
+            raise EasyBuildError("Unable to find 'ldconfig' in $PATH = %s " +
+                                 "nor in /sbin or /usr/sbin" % os.environ.get('PATH', ''))
+
         # Run ldconfig to create missing symlinks in the stubs directory (libcuda.so.1, etc)
-        run_cmd("ldconfig -N " + os.path.join(self.installdir, 'lib64', 'stubs'))
+        cmd = " ".join(ldconfig, '-N', os.path.join(self.installdir, 'lib64', 'stubs'))
```

Comment:
@akesandgren This `join` is incorrect, fixed in https://github.com/akesandgren/easybuild-easyblocks/pull/9